templates/security/register.html.twig

```diff
@@ -0,0 +1,88 @@
+    <!--[if lt IE 9]>
+    <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+    <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+</head>
+
+<body>
+
+<div class="container">
+    <div class="row">
+        <div class="col-md-4 col-md-offset-4">
+            <div class="login-panel panel panel-default">
+                <div class="panel-heading">
+                    <h3 class="panel-title">Register user</h3>
+                </div>
+                <div class="panel-body">
+
+                    {#{% if error %}#}
+                        {#<div>{{ error.messageKey|trans(error.messageData, 'security') }}</div>#}
+                    {#{% endif %}#}
+                    {##a94442#}
+
+                    {#{% if form.email.vars.errors|length %} has-error{% endif %}#}
+
+
+
+                    {{ form_start(form) }}
+                    <div class="form-group {% if form.email.vars.errors|length %} has-error{% endif %}">
+                        {{ form_widget(form.email, {'attr': {'class': 'form-control'}}) }}
+                    </div>
+                    {% if form.email.vars.errors|length %}
+                        <div class="message-warning">
+                            {{ form_errors(form.email) }}
+                        </div>
+                    {% endif %}
+                    <div class="form-group {% if form.password.password.vars.errors|length %} has-error{% endif %} ">
+                        {{ form_widget(form.password.password, {'attr': {'class': 'form-control'}}) }}
+                    </div>
+                    <div class="form-group {% if form.password.password.vars.errors|length %} has-error{% endif %}">
+                        {{ form_widget(form.password.repeatPassword, {'attr': {'class': 'form-control'}}) }}
+                    </div>
+                    {% if form.password.password.vars.errors|length %}
+                        <div class="message-warning">
+                            {{ form_errors(form.password.password) }}
+                        </div>
+                    {% endif %}
+                    <div class="form-group {% if form.vatIdNumber.vars.errors|length %} has-error{% endif %}">
+                        {{ form_widget(form.vatIdNumber, {'attr': {'class': 'form-control'}}) }}
+                    </div>
+                    {% if form.vatIdNumber.vars.errors|length %}
+                        <div class="message-warning">
+                            {{ form_errors(form.vatIdNumber) }}
+                        </div>
+                    {% endif %}
+                    {{ form_widget(form.submit, {'attr': {'class': 'btn btn-lg btn-success btn-block'}}) }}
+                    {{ form_end(form) }}
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<script src="//netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js"></script>
+</body>
+</html>
```

Comment:
Yeah, but we can do separate base template for login and register actions (seems to be quite simillar)